src/SDK/Subscriber/Event/EndSpanEvent.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Subscriber\Event;
+
+use OpenTelemetry\SDK\Subscriber\EventInterface;
+
+class EndSpanEvent implements EventInterface
+{
+    private array $array ;
```

Comment:
Please use a different name that is more descriptive.
Naming things in software development is hard, but naming things after their type doesn't tell what it's supposed to do.